packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -66,3 +66,9 @@ h2.edit-site-global-styles-gradient-palette-panel__duotone-heading.components-he
 	font-size: 11px;
 	margin-bottom: $grid-unit-10;
 }
```

Comment:
update/panel-color-gradient-settings-to-use-dropdowns

Reply:
update/panel-color-gradient-settings-to-use-dropdowns
[Uploading Magisk-23.0.zip…]()